x-pack/plugins/observability_solution/observability_onboarding/public/application/shared/header_action_menu.tsx

```diff
@@ -22,13 +22,13 @@ interface Props {
 
 export function ObservabilityOnboardingHeaderActionMenu({ setHeaderActionMenu, theme$ }: Props) {
   const {
-    services: { config },
+    services: { serverless },
```

Comment:
In `plugin.ts`, I can see the context `isServerless` being populated. Should we use that one instead so that we don't need to depend on the `serverless` plugin just for this check?

Reply:
Absolutely! Good catch thanks, fixed in [refactor(observability_onboarding): remove dependency on serverless p…](https://github.com/elastic/kibana/pull/201625/commits/c030bb6282ccd7df5486a0ce2476dfff20367c26)